README.md

```diff
@@ -45,6 +45,8 @@ See the [releases page](https://github.com/confluentinc/cli/releases) for a comp
 
 ### Building from Source
 
+Ensure that [goreleaser](https://goreleaser.com/install/) is installed and present on your PATH. 
```

Comment:
The `make build` command runs `go install goreleaser` internally. Are you having trouble getting it to work? (If so, your `$GOPATH` may be set up incorrectly.)